onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlDFT.h

```diff
@@ -1081,8 +1121,11 @@ class GpuDFTOperatorFactory : public WRL::Base<IMLOperatorKernelFactory>
         {
             MLOperatorEdgeDescription { MLOperatorEdgeType::Tensor, (uint64_t)MLOperatorTensorDataType::Float16 },
             MLOperatorEdgeDescription { MLOperatorEdgeType::Tensor, (uint64_t)MLOperatorTensorDataType::Float },
-            //MLOperatorEdgeDescription { MLOperatorEdgeType::Tensor, (uint64_t)MLOperatorTensorDataType::Double },
         };
+        if (version == 20)
+        {
+            t1AllowedEdges.push_back(MLOperatorEdgeDescription { MLOperatorEdgeType::Tensor, (uint64_t)MLOperatorTensorDataType::Double });
+        }
```

Comment:
why this? wasnt double supported even in 17?

Reply:
It was commented out in 17 so I added a check, but you're right, will make the change